### DISCUSSION TITLE
DescriptionList (or NameValueList) component?

### DISCUSSION THREAD

**Discussion by fengye87** [mantinedev/mantine#6821] [Feature requests]
Could we have a component to display key-value data nicely? Something like:

- https://tabler.io/docs/components/datagrid
- https://v2.grommet.io/namevaluelist

**Comment by Kenzo-Wada** [comment]
I don't think Mantine has it... I think it's something that you can easily create using simple components such as Grid, Flex, Stack and Text!

**Reply by fengye87** [reply]
It surely can. But I think it's a basic component, there's a `<dl>` tag in HTML standard. It would be nice to have it in the lib

**Reply by SteveDowsett** [reply]
I agree, this is something that is really common in apps, sure can roll your own, but would be nice to have!
